src/ontology/uberon-edit.obo

```diff
@@ -224404,6 +224404,18 @@ relationship: dc-contributor https://orcid.org/0000-0002-5480-427X
 relationship: in_taxon NCBITaxon:9669 ! Mustela putorius furo
 relationship: part_of UBERON:0002104 ! visual system
 
+[Term]
+id: UBERON:8440072
+name: lateral tegmental nucleus
+def: "A nucleus of the brain that is embedded laterally in the pontine central gray of the pontine tegmentum, it is defined on the basis of inputs from the lateral hypothalamic area and central amygdalar nucleus, it is located ventral to the locus ceruleus, dorsal to Barrington's nucleus, lateral to the laterodorsal tegmental nucleus, and medial to the midbrain trigeminal nucleus." [neuronames:1957]
+xref: MBA:283
+xref: neuronames:1957
+is_a: UBERON:0002308 ! nucleus of brain
+relationship: dc-contributor http://orcid.org/0000-0001-7258-9596
+relationship: dorsal_to UBERON:0007632 ! Barrington's nucleus
+relationship: part_of UBERON:0000988 ! pons
```

Comment:
this can be more specific. The text def mentions the two bolded terms:

* [] UBERON:0001016 ! nervous system
    * [p] UBERON:0001017 ! central nervous system
        * [p] UBERON:0000955 ! brain
            * [p] UBERON:0002028 ! hindbrain
                * [p] UBERON:0001895 ! metencephalon
                    * [p] UBERON:0000988 ! pons
                        * [p] **UBERON:0003023 ! pontine tegmentum**
                            * [p] UBERON:0002573 ! pontine reticular formation
                                * [p] **UBERON:0002968 ! central gray substance of pons**

I assume you are basing the structure on MBA which has this as a sibling of the central gray: 

* [] UBERON:0002616
    * [i] **MBA:283 ! lateral tegmental nucleus**
    * [i] MBA:997 ! root
        * [p] MBA:8 ! basic cell groups and regions
            * [p] MBA:343 ! brain stem
                * [p] MBA:1065 ! hindbrain
                    * [p] MBA:771 ! pons
                        * [p] MBA:987 ! pons, motor related
                            * [p] **MBA:283 ! lateral tegmental nucleus**
                            * [p] **MBA:898 ! pontine central gray**

obviously everyone draws lines differently and global consistency is impossible but I suggest taking the neuronames description as canonical for now, in the absence of input from a neuroanatomy SME

Reply:
I think @patrick-lloyd-ray as our neuroanatomy SME :) Given MBA has pontine central gray (which maps to central gray substance of pons), which is sibling to lateral tegmental nucleus, and the reason im adding that is so that we can map it mba, I'll put it as part of 'pontine tegmentum' instead for safety. I'll edit the def accordingly too. That way it will follow both neuronames and mba

Reply:
> I assume you are basing the structure on MBA 

Trying to at least. unfortunately the lateral tegmental nucleus is not shown in the atlas (as in not drawn out), which makes verifying the above a bit tricky too.

Reply:
It's accurate to place it as part of 'pontine tegmentum'.